tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/SecurityNotificationTest.java

```diff
@@ -155,6 +163,8 @@ public void testCONSUMER_CREATED() throws Exception {
       Assert.assertEquals("guest", notifications[0].getObjectProperty(ManagementHelper.HDR_VALIDATED_USER).toString());
       Assert.assertEquals(address.toString(), notifications[0].getObjectProperty(ManagementHelper.HDR_ADDRESS).toString());
       Assert.assertEquals(SimpleString.toSimpleString("unavailable"), notifications[0].getSimpleStringProperty(ManagementHelper.HDR_CERT_SUBJECT_DN));
+      Assert.assertTrue(notifications[0].getTimestamp() >= start);
+      Assert.assertTrue((Long) notifications[0].getObjectProperty(ManagementHelper.HDR_NOTIFICATION_TIMESTAMP) >= start);
```

Comment:
shouldn't we assert for == .getTimestamp()? given the way it's implemented?

Reply:
Are you saying we should assert `getTimestamp() == start` or something else? Both timestamps should be the same and both should be *after* `start`. I'm just asserting the latter.

Reply:
i think the point here, is shouldnt the test be check timestamp >= start, and check timestamp aloing with hdr notif timestamp field are the same.

current logic in test doesnt check both are the same.

Reply:
Fair enough. I updated the tests to assert both timestamps are equal.